draft-ietf-curdle-gss-keyex-sha2.xml

```diff
@@ -88,20 +89,22 @@
   <middle>
     <section title="Introduction">
       <t><xref target="RFC4462">SSH GSS-API Methods</xref> allows the use of
-      GSSAPI for authentication and key exchange in SSH. It defines three
-      exchange methods all based on DH groups and SHA-1. This document
-      updates RFC4462 with new methods intended to support environments that
-      desire to use the SHA-2 cryptographic hash functions.</t>
+      <xref target="RFC2743">GSSAPI</xref> for authentication and key exchange
+      in SSH. It defines three exchange methods all based on DH groups and
+      SHA-1. This document updates RFC4462 with new methods intended to support
```

Comment:
shouldn't RFC4462 be in xref here?

Reply:
nobody request it so I am ok leaving as is, it's not like links to RFC4462 are not plentiful in the document already :-)